### PR TITLE
add additional facts

### PR DESCRIPTION
**Pull request by b4ldr**
Hi i took a look at the get_facts script and added a few fixes as it didn't seem to work.  I have also created a new script called `generate_facts.sh` which just spins up each machine one at a time as i dont have the disk space to store all vms.   I also changed the logic so that if a file already exists it is not updated which helps keep the git commit smaller.  Other then that all fact files where created by running 
```
cd facts
./generate_facts.sh
```

### CODE REVIEW

File: facts/1.6/linuxmint-18-x86_64.facts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  "type": "Other",
+  "memorysize": "488.41 MB",
+  "memoryfree": "196.36 MB",
+  "swapsize": "510.00 MB",
+  "swapfree": "371.64 MB",
+  "memorytotal": "488.41 MB",
+  "netmask": "255.255.255.0",
+  "network_enp0s3": "10.0.2.0",
+  "network_lo": "127.0.0.0",
+  "operatingsystemrelease": "stretch/sid",
```
Comment (mmoll) [inline facts/1.6/linuxmint-18-x86_64.facts]: that Debian 9 image was not fully updated to a 9.x release.

**Comment by mmoll** [comment]
see GH-52 for problems with Debian/stretch facts.

**Comment by b4ldr** [comment]
@mmoll i have updated the vagrant image used for debian-9 and believe that the get_facts script is now producing correct facts

**Comment by mmoll** [comment]
@b4ldr this is looking good 👍

**Comment by mcanevet** [comment]
@b4ldr please remove `facts/.Vagrantfile.swp` from your commit

**Comment by b4ldr** [comment]
@mcanevet corrected, also updated the gitignore file

**Comment by mmoll** [comment]
this needs a rebase now

**Comment by b4ldr** [comment]
rebased

**Comment by b4ldr** [comment]
is anything holding this back?

**Comment by mcanevet** [comment]
@b4ldr please split this PR in different commit in case we have to revert one part.
Don't mix scripts and fact changes in the same commit.

**Comment by b4ldr** [comment]
ok i have split out the vagrant files into #59, if you accept that pull then you can generate this pull request you self by running  `./facts/generate_facts.sh`

**Pull request closed without merging**